spec/helpers_spec.cr

```diff
@@ -265,7 +265,7 @@ describe "Macros" do
       Kemal.config.serve_static.should eq false
     end
 
-    it "should disble enable gzip and dir_listing" do
+    it "should disable enable gzip and dir_listing" do
```

Comment:
it "should enable gzip and dir_listing" do ?

Reply:
Yes, please fix 👍